### PR TITLE
LPD-26435 Close active menu when clicking another menu

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
https://liferay.atlassian.net/browse/LPD-26435
https://liferay.atlassian.net/browse/LPP-54153

Using liferay-ui:icon-menu, the menus will not be hidden if another menu is opened leaving two menu on screen. We need to close the active menu when a new menu is clicked.

Please let me know if there are any questions or comments about this.
Thank you!

### CODE REVIEW

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

File: modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts
Comment (markocikos) [inline modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts:1]: [nit] File name should reflect the component or module that we are testing, not the test itself.

File: modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {systemSettingsPageTest} from '../../fixtures/systemSettingsPageTest';
+
+export const test = mergeTests(loginTest(), systemSettingsPageTest);
+
+test('@LPD-26435 Close active menu', async ({page}) => {
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts:13]: [nit] This is not very descriptive of what we are testing. We are not testing closing of active menu, it's just a step that happens in the test.

File: modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		await contentAndData.click();
+
+		const wiki = page.getByRole('menuitem', {name: 'Wiki'});
+
+		await wiki.waitFor({state: 'visible'});
+		wiki.click();
+	});
+
+	await test.step('Create new wiki', async () => {
+		const newWikiButton = page.getByRole('link', {name: 'Add Wiki'});
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts:35]: We should avoid adding tests on pages of other product teams. There are two ways around this:
1. We can add this test into the relevant project, not new `frontend-js-aui-web`.
2. We can build a sample module with this case and build a test for it.

I don't think we have a sample module for AUI stuff, and we probably don't want to create one. With that in mind, (1) is probably the right way to go in this case. We should add this to existing `wiki` tests, if they exist, or add them. We should also inform responsible team and ask for a review. @dsanz @herrardo what do you think?

**Review by markocikos — COMMENTED** [review]
@fortunatomaldonado see comments inline

**Comment by fortunatomaldonado** [comment]
ci:test:sf

File: modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts
Comment (fortunatomaldonado) [reply modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts:1]: Got it!

File: modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts (hunk 1 above)
Comment (fortunatomaldonado) [reply modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts]: Got it, changed!

File: modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts (hunk 2 above)
Comment (fortunatomaldonado) [reply modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts]: I created wiki-web test. If this gets the go ahead I will send to Lima which I believe is responsible for wiki-web.

File: modules/test/playwright/tests/wiki-web/wikiWeb.spec.ts
Comment (markocikos) [inline modules/test/playwright/tests/wiki-web/wikiWeb.spec.ts:1]: I'd just call this `wiki.spec.ts`, `Web` is not necessary. The naming here is ultimately for the product team, if they want to make it more specific for some section of `wiki` app.

**Review by markocikos — APPROVED** [review]
@fortunatomaldonado a minor non-blocking nit inline, otherwise LGTM!

**Comment by fortunatomaldonado** [comment]
FWD: https://github.com/liferay-content-management/liferay-portal/pull/5481

**Pull request closed without merging**